nautobot/core/tables.py

```diff
@@ -557,7 +557,7 @@ def render(self, *, bound_column, record, value):  # pylint: disable=arguments-d
                 related_record = related_records[0]
         url = reverse(self.viewname, kwargs=self.view_kwargs)
         if self.url_params:
-            url += "?" + urlencode({k: getattr(record, v) for k, v in self.url_params.items()})
+            url += "?" + urlencode({k: (getattr(record, v) or "") for k, v in self.url_params.items()})
```

Comment:
This won't have the expected UX in this specific case, will it? It'll lead to an unfiltered list of "all" records, which doesn't seem like it'd be that useful as a list of "related" Prefixes to a given Wireless Network.